packages/components/fields/money-field/src/money-field.tsx

```diff
@@ -0,0 +1,284 @@
+import {
+  Component,
+  isValidElement,
+  type ReactElement,
+  type ReactNode,
+} from 'react';
+import { type Props as ReactSelectProps } from 'react-select';
+import {
+  filterDataAttributes,
+  createSequentialId,
+  getFieldId,
+  warning,
+} from '@commercetools-uikit/utils';
+import Constraints from '@commercetools-uikit/constraints';
+import Spacings from '@commercetools-uikit/spacings';
+import FieldLabel from '@commercetools-uikit/field-label';
+import MoneyInput from '@commercetools-uikit/money-input';
+import FieldErrors from '@commercetools-uikit/field-errors';
+import currencies from './currencies.json';
+
+type TFieldErrors = Record<string, boolean>;
+
+type TErrorRenderer = (key: string, error?: boolean) => ReactNode;
+
+const sequentialId = createSequentialId('money-field-');
+
+const hasErrors = (errors?: TFieldErrors) =>
+  errors && Object.values(errors).some(Boolean);
+
+type TCurrencyCode = keyof typeof currencies;
```

Comment:
Instead of duplicating the currencies JSON, I think it makes sense to expose the `TCurrencyCode` type from the `money-input` package and use that.